Documentation/ceph-upgrade.md

```diff
@@ -126,22 +120,21 @@ In a healthy Rook cluster, the operator, the agents and all Rook namespace pods
 `Running` state and have few, if any, pod restarts. To verify this, run the following commands:
 ```sh
 kubectl -n $ROOK_SYSTEM_NAMESPACE get pods
-kubectl -n $ROOK_NAMESPACE get pod
+kubectl -n $ROOK_NAMESPACE get pods
 ```
 
-If pods aren't running or are restarting due to crashes, you can get more information with
-and  for the affected pods by trying the commands below.
-* `kubectl -n <namespace> describe pod`
-* `kubectl -n <namespace> logs --previous`
+If pods aren't running or are restarting due to crashes, you can get more information for the
```

Comment:
What if we point them to the rook troubleshooting guide instead of trying to provide these commands here?